codewind-che-sidecar/scripts/kube/codewind_template.yaml

```diff
@@ -85,7 +85,7 @@ spec:
           secretName: REGISTRY_SECRET_PLACEHOLDER
       containers:
       - name: codewind-pfe
-        image: sys-mcs-docker-local.artifactory.swg-devops.com/codewind-pfe-amd64:latest
+        image: ibmcom/codewind-pfe-amd64:latest
```

Comment:
Don't change the Codewind PFE image to Dockerhub yet. Since our latest builds of PFE are still getting pushed to Artifactory, we want to keep the Codewind deployment pointing to Artifactory on the sidecar master branch.

Reply:
ok